src/main/java/task/TaskManager.java

```diff
@@ -0,0 +1,53 @@
+package task;
+
+public class TaskManager {
+    static protected Task[] list;
```

Comment:
Should this be named "taskList" instead just "list"? "taskList" seems to be in sync with your counter "taskCount" and so on.